test/K6/src/api/platform/authorization/maskinporten.js

```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
   return http.post(endpoint, bodystring, params);
 }
 
+/**
+ * GET call to get maskinportenschemas that have been received by the current party
+ * @param {*} altinnToken personal token for DAGL
+ * @param {*} offeredByOrganizationNumber the organization number for the offeredby party
+ * @param {*} resourceid the id of the resource to delegate
+ * @param {*} attributeId the attribute id for the receiver of the schema. 'urn:altinn:partyid' or 'urn:altinn:organizationnumber'
+ * @param {*} attributeValue the receiver's partyid or organization number
+ */
+export function postMaskinportenSchemaOrgNoInHeader(altinnToken, offeredByOrganizationNumber, resourceid, attributeId, attributeValue) {
+  var endpoint = config.buildMaskinPorteSchemaUrls('organization', 'maskinportenschema');
+  console.log(endpoint)
+  var params = header.buildHeaderWithRuntimeOrgNumberAndJson(altinnToken, 'personal', offeredByOrganizationNumber);
```

Comment:
what's the "personal" parameter?

Reply:
I saw that the 'personal' parameter was used to choose an endpoint in token-generator.js, and didn't dig any deeper when I saw it here. I see now that it's unused, so I removed it.